src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS = 10;
+    private static final int NUMBER_FIGURE = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
    String color = colorSupplier.getRandomColor();
```

pass it to each type of figure